cli/lib/kontena/cli/stacks/build_command.rb

```diff
@@ -11,6 +12,8 @@ class BuildCommand < Kontena::Command
     option ['--no-cache'], :flag, 'Do not use cache when building the image', default: false
     option ['--no-push'], :flag, 'Do not push images to registry', default: false
     option ['--no-pull'], :flag, 'Do not attempt to pull a newer version of the image', default: false
+    option ['--[no-]sudo'], :flag, 'Run docker using sudo', hidden: Kontena.on_windows?, environment_variable: 'KONTENA_SUDO', default: false
```

Comment:
I was hoping someone would ask :)

If you have `KONTENA_SUDO=true` but want to run the command without sudo, you would use `--no-sudo` which overrides the environment variable.